server/db/driver/pg/orders_online_test.go

```diff
@@ -432,15 +440,51 @@ func TestFlushBook(t *testing.T) {
 	if err != nil {
 		t.Errorf("ExecutedCancelsForUser failed: %v", err)
 	}
-	if len(coids) != 1 {
-		t.Fatalf("got %d cancels, expected 1", len(coids))
+	// ExecutedCancelsForUser should not find the (exempt) cancels created by
+	// FlushBook.
+	if len(coids) != 0 {
+		t.Fatalf("got %d cancels, expected 0", len(coids))
 	}
-	if len(targets) != 1 {
-		t.Fatalf("got %d cancel targets, expected 1", len(targets))
+	if len(targets) != 0 {
+		t.Fatalf("got %d cancel targets, expected 0", len(targets))
 	}
 
-	if targets[0] != lo.ID() {
-		t.Fatalf("cancel order is targeting %v, expected %v", targets[0], lo.ID())
+	// Query for the revoke associated cancels without the exemption filter.
+	cancelTableName := fullCancelOrderTableName(archie.dbName, mktInfo.Name, false)
+	stmt := fmt.Sprintf(internal.SelectRevokeCancels, cancelTableName)
+	rows, err := archie.db.QueryContext(context.Background(), stmt, lo.User(), orderStatusRevoked, 25)
+	if err != nil {
+		t.Fatalf("QueryContext failed: %v", err)
+	}
```

Comment:
Note that there is no function either exported or unexported to load these exempt/uncounted cancels because they are there just to have a permanent record of the order revocation event.  Some administrative functions may want to query for these in the future however.